esmvalcore/config-user.yml

```diff
@@ -45,19 +72,30 @@ profile_diagnostic: false
 # For site-specific entries, see below
 # Comment out these when using a site-specific path
 rootpath:
-  default: ~/climate_data
+  CMIP3: [~/cmip3_inputpath1, ~/cmip3_inputpath2]
+  CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
+  CMIP6: [~/cmip6_inputpath1, ~/cmip6_inputpath2]
+  OBS: ~/obs_inputpath
+  OBS6: ~/obs6_inputpath
+  obs4MIPs: ~/obs4mips_inputpath
+  ana4mips: ~/ana4mips_inputpath
+  native6: ~/native6_inputpath
+  RAWOBS: ~/rawobs_inputpath
+  default: ~/default_inputpath
```

Comment:
What was the reason for changing this back to the old default from before #1217? I liked the new default value here because it would work for the data that gets automatically downloaded with the ``--offline=False`` method.

I think the old example was confusing because it contains a mix of lists of paths and paths.

Reply:
To be honest I didn't really think about this and used the options that were use in ESMValTool. Showcasing the possibility to use multiple paths and also list all available projects seemed like a good idea. If a user wants to use this file with automatic downloads they need to change the default value of `offline` anyway, and in the file there is a comment:

```
Uncomment the lines below to locate data that has been automatically
# downloaded from ESGF (using "offline: false")
```

I'm also fine with the old one, but don't really see the use. For me this is an example file which needs to be altered anyway.

Reply:
The idea behind the changes I made in #1217 was to finally make it possible to run the tool for the first time without editing the example config-user.yml, as documented [here](https://docs.esmvaltool.org/en/latest/quickstart/running.html#running-a-recipe). That failed slightly because people objected to automatically downloading the required data, but if you specify `--offline=False` on the command line it still works.

Showcasing per project paths and lists of paths is also good, but may we could add a few more commented-out examples instead to showcase that?

Reply:
Sure, will add that to #1457